src/main.js

```diff
@@ -88,33 +45,14 @@ if (args.dev) {
 
 // Add logging support, based on
 //   https://cloud.google.com/nodejs/getting-started/logging-application-events
-winston.level = args.log_level;
 const appWithLogging = express();
 
-function getTransports(json, colorize) {
-    const transports = [];
-    if (!args.dev) {
-        transports.push(new StackdriverTransport());
-    }
-    transports.push(new winston.transports.Console({
-        json,
-        colorize,
-    }));
-    return transports;
-}
-
-appWithLogging.use(expressWinston.logger({      // request logging
-    transports: getTransports(false, args.dev), // colorize for dev, not prod
-    expressFormat: true,
-    meta: false,
-}));
-appWithLogging.use(expressWinston.errorLogger({      // error logging
-    transports: getTransports(true, args.dev), // colorize for dev, not prod
-}));
+appWithLogging.use(logging.middleware.requestLogger);
+appWithLogging.use(logging.middleware.errorLogger);
 appWithLogging.use(app);
 
 const server = appWithLogging.listen(port, () => {
     const host = server.address().address;
     const port = server.address().port;
-    logging.info('react-render-server running at http://%s:%s', host, port);
+    logging.info(`react-render-server running at http://%s:%s`, host, port);
```

Comment:
I had used interpolation to fix this message until I discovered the `splat()` formatter that allows for the %s approach. I must've forgotten to put the quotes back the way they were.